### DISCUSSION TITLE
Ubuntu for SO 2.4.10

### DISCUSSION THREAD

**Discussion by its-a-me-mario-the-og** [Security-Onion-Solutions/securityonion#11299] [Unsupported Versions] [answered]
Hey SO team, I know the doc has Ubuntu has being one OS that can run SO but has to be manually installed. Also I understand that their might be more problems with the Ubuntu version and everything. But will there be a manual installation guide for people who don't want to use the Oracle ISO?

**Comment by jaypfe** [comment, marked as answer]
There is one in their docs!
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/107776997/112577d6-56d8-4884-af32-3bfb792a584b)

https://docs.securityonion.net/en/2.4/installation.html
